mempool/src/shared_mempool.rs

```diff
@@ -165,6 +166,22 @@ pub struct TransactionExclusion {
     pub sequence_number: u64,
 }
 
+/// Mempool's subscription to reconfiguration events
+pub struct MempoolReconfigSubscription {
+    sender: futures::channel::mpsc::Sender<usize>,
```

Comment:
just for context, what are you expecting will be the payload type?

Reply:
I'm just using `usize` as a dummy placeholder - but I can make it `Vec<ContractEvent>` to make it more realistic

Reply:
no worries, feel free to do it in a later PR